mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -529,7 +529,7 @@ class LinearRegressionSummary private[regression] (
     val predictionCol: String,
     val labelCol: String,
     val model: LinearRegressionModel,
-    val diagInvAtWA: Array[Double]) extends Serializable {
+    private val diagInvAtWA: Array[Double]) extends Serializable {
```

Comment:
@jkbradley I vote for private at present. Users will care about ```coefficientStandardErrors```, ```tValues``` and ```pValues``` which are calculated by ```diagInvAtWA``. Meanwhile users can not get ```diagInvAtWA``` at R by public interface. @mengxr